editor/plugins/script_text_editor.cpp

```diff
@@ -1304,6 +1304,12 @@ void ScriptTextEditor::_edit_option(int p_op) {
 			code_editor->get_text_editor()->insert_text_at_caret(String("\n").join(results));
 			code_editor->get_text_editor()->end_complex_operation();
 		} break;
+		case EDIT_INSERT_CARET_BELOW: {
+			code_editor->get_text_editor()->insert_caret_at_carets(true);
+		} break;
+		case EDIT_INSERT_CARET_ABOVE: {
+			code_editor->get_text_editor()->insert_caret_at_carets(false);
+		} break;
```

Comment:
The only reason I added these to the edit menu is so that users see that the actions exists. If they are not in the edit menu, the user will only notice that they exist if he looks at the shortcut settings. If this reason does not suffice, I can remove them.